package-lock.json

```diff
@@ -865,11 +875,6 @@
         "node": ">=10.0.0"
       }
     },
-    "node_modules/@duckduckgo/autoconsent": {
```

Comment:
please ensure, the autoconsent is included once in the package-lock.json.

if you don't find a better way with npm cli, then you can always update package-lock by hand. in this case it should be easy

Reply:
Successfully optimized the tree.